doc/js_tutorials/js_setup/js_nodejs/js_nodejs.markdown

```diff
@@ -227,52 +227,127 @@ In the browser, this filesystem is emulated in memory while in Node.js there's a
+const src = cv.imread(image);
+let srcBGR = new cv.Mat();
+cv.cvtColor(src, srcBGR, cv.COLOR_RGBA2BGR);
+
+// Load the deep learning model file. Notice how we reference local files using relative paths just
+// like we normally would do
+let netDet = new cv.FaceDetectorYN("./face_detection_yunet_2023mar.onnx", "", new cv.Size(320, 320), 0.9, 0.3, 5000);
+netDet.setInputSize(new cv.Size(src.cols, src.rows));
+let out = new cv.Mat();
+netDet.detect(srcBGR, out);
+
+let faces = [];
+for (let i = 0, n = out.data32F.length; i < n; i += 15) {
+  let left = out.data32F[i];
+  let top = out.data32F[i + 1];
+  let right = (out.data32F[i] + out.data32F[i + 2]);
+  let bottom = (out.data32F[i + 1] + out.data32F[i + 3]);
+  left = Math.min(Math.max(0, left), src.cols - 1);
+  top = Math.min(Math.max(0, top), src.rows - 1);
+  right = Math.min(Math.max(0, right), src.cols - 1);
+  bottom = Math.min(Math.max(0, bottom), src.rows - 1);
+
+  if (left < right && top < bottom) {
+    faces.push({
+      x: left,
+      y: top,
+      width: right - left,
+      height: bottom - top,
+      x1: out.data32F[i + 4] < 0 || out.data32F[i + 4] > src.cols - 1 ? -1 : out.data32F[i + 4],
+      y1: out.data32F[i + 5] < 0 || out.data32F[i + 5] > src.rows - 1 ? -1 : out.data32F[i + 5],
+      x2: out.data32F[i + 6] < 0 || out.data32F[i + 6] > src.cols - 1 ? -1 : out.data32F[i + 6],
+      y2: out.data32F[i + 7] < 0 || out.data32F[i + 7] > src.rows - 1 ? -1 : out.data32F[i + 7],
+      x3: out.data32F[i + 8] < 0 || out.data32F[i + 8] > src.cols - 1 ? -1 : out.data32F[i + 8],
+      y3: out.data32F[i + 9] < 0 || out.data32F[i + 9] > src.rows - 1 ? -1 : out.data32F[i + 9],
+      x4: out.data32F[i + 10] < 0 || out.data32F[i + 10] > src.cols - 1 ? -1 : out.data32F[i + 10],
+      y4: out.data32F[i + 11] < 0 || out.data32F[i + 11] > src.rows - 1 ? -1 : out.data32F[i + 11],
+      x5: out.data32F[i + 12] < 0 || out.data32F[i + 12] > src.cols - 1 ? -1 : out.data32F[i + 12],
+      y5: out.data32F[i + 13] < 0 || out.data32F[i + 13] > src.rows - 1 ? -1 : out.data32F[i + 13],
+      confidence: out.data32F[i + 14]
+    })
+  }
+}
+out.delete();
+
+faces.forEach(function(rect) {
+  cv.rectangle(src, {x: rect.x, y: rect.y}, {x: rect.x + rect.width, y: rect.y + rect.height}, [0, 255, 0, 255]);
+  if(rect.x1>0 && rect.y1>0)
+    cv.circle(src, {x: rect.x1, y: rect.y1}, 2, [255, 0, 0, 255], 2)
+  if(rect.x2>0 && rect.y2>0)
+    cv.circle(src, {x: rect.x2, y: rect.y2}, 2, [0, 0, 255, 255], 2)
+  if(rect.x3>0 && rect.y3>0)
+    cv.circle(src, {x: rect.x3, y: rect.y3}, 2, [0, 255, 0, 255], 2)
+  if(rect.x4>0 && rect.y4>0)
+    cv.circle(src, {x: rect.x4, y: rect.y4}, 2, [255, 0, 255, 255], 2)
+  if(rect.x5>0 && rect.y5>0)
+    cv.circle(src, {x: rect.x5, y: rect.y5}, 2, [0, 255, 255, 255], 2)
+});
+
+const canvas = createCanvas(image.width, image.height);
+cv.imshow(canvas, src);
+console.log('The result is saved.')
+writeFileSync('output3.jpg', canvas.toBuffer('image/jpeg'));
```

Comment:
> m.b. writeFileSync?

Reply:
Sorry for typo. I mean print log after write when it's complete.

Reply:
Done.